tests/server/util.c

```diff
@@ -671,7 +671,7 @@ static SIGHANDLER_T set_signal(int signum, SIGHANDLER_T handler,
   sa.sa_handler = handler;
   sigemptyset(&sa.sa_mask);
   sigaddset(&sa.sa_mask, signum);
-  sa.sa_flags = restartable? SA_RESTART: 0;
+  sa.sa_flags = restartable ? SA_RESTART: 0;
```

Comment:
These asymmetric ones look strange to me now, where there are spaces on both sides of `?` but only on one side of `:`. Looks good to me other than that.

Reply:
It's not intentional, but checksrc does not check the colons (yet).